### DISCUSSION TITLE
anti kick issue

### DISCUSSION THREAD

**Discussion by ominouscan** [winstxnhdw/lc-hax#486] [General]
anti kick only works sometimes becuase what happens is when i join a server and they kick me and i automatically rejoin it says "the ship has already taken off" or landed something like that, so how do i fix this so that i cant be kicked in any server i join becuase this happens a lot and i know that they didnt land the ship.

**Comment by winstxnhdw** [comment]
Just force a rejoin with Shift + F5?

**Reply by ominouscan** [reply]
nope still says ship has landed when i shift + F5 think its broken also it happens to my friend who also has the cheat

**Reply by winstxnhdw** [reply]
I usually just do it a few times until I get it.

**Reply by Totoqoe** [reply]
The issue is quite literal, you're unable to join after the ship lands so you have to wait for the ship to orbit again if the host has a late join mod. MoreCompany used to have the lobby marked as join able at all times which is why the rejoin *used* to work in the middle of a mission.
https://github.com/notnotnotswipez/MoreCompany/pull/42

**Reply by Viktor7774564** [reply]
how to enable anti-kick?

**Reply by winstxnhdw** [reply]
Backslash.
